src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. this will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
+  </data>
+  <data name="IntellisenseBuildFailedTitle" xml:space="preserve">
+    <value>Project loading failed.</value>
+  </data>
+  <data name="IntellisenseBuildFailedDescription" xml:space="preserve">
+    <value>To see what caused the issue, please try below.
```

Comment:
I agree that we should have this text somewhere other than just the description so that it's searchable by a search engine. We already use fwlinks when we have the precise link instead of depending on bing and so using fwlink makes sense. 

If this is the description then this will only appear hidden under the chevron - correct? The error list is not going to be flooded with these messages but they are available if someone needs them? That might be fine.